microcosm_pubsub/result.py

```diff
@@ -24,13 +26,13 @@ class MessageHandlingResultType(Enum):
     EXPIRED = MessageHandlingResultTypeInfo(name="EXPIRED", level=WARNING)
 
     # Messaging handling failed.
-    FAILED = MessageHandlingResultTypeInfo(name="FAILED", level=WARNING, exc_info=True)
+    FAILED = MessageHandlingResultTypeInfo(name="FAILED", level=WARNING, exc_info=True, retry=True)
 
     # Message was not handled.
     IGNORED = MessageHandlingResultTypeInfo(name="IGNORED", level=DEBUG)
 
     # Messaging handling was intentionally retried.
-    RETRIED = MessageHandlingResultTypeInfo(name="RETRIED", level=INFO)
+    RETRIED = MessageHandlingResultTypeInfo(name="RETRIED", level=INFO, retry=True)
```

Comment:
Now we have a reason the rename `RETRIED` (ping @GabeLambda :))

Reply:
I could also rename `retry` to something else...

Reply:
Like `should_nack=True`?
I think we should keep `retry` - we do use the `retry` terminology  (`retry_timeout_seconds`, not `nack_timeout_seconds`)

Reply:
It's either that, or go back to using the nack terminology (i.e. NACKED).